assets/less/content/general.less

```diff
@@ -101,18 +101,98 @@ h1 .note {
 
 blockquote {
   font-style: italic;
-  margin: .5em 0;
-  padding: .25em 1.5em;
   border-left: 3px solid @gray;
-  display: inline-block;
+  position: relative;
+  margin: 1.5625em 0;
+  padding: 0 1.2rem;
+  overflow: auto;
+  background-color: #f6f6f6;
+  border-radius: 3px;
 
-  *:first-child {
-    padding-top: 0;
-    margin-top: 0;
+  &.warning,
+  &.error,
+  &.info,
+  &.neutral {
+    color: #000;
+    border-radius: 10px;
+    border-left: 0;
+  }
+
+  &.warning {
+    background-color: #fff7ed;
+  }
+
+  &.error {
+    background-color: #fdeeec;
+  }
+
+  &.info {
+    background-color: #e9f5fe;
+  }
+
+  &.neutral {
+    background-color: #e2e8ef;
+  }
+
+  h3.warning,
+  h3.error,
+  h3.info,
+  h3.neutral,
+  h4.warning,
+  h4.error,
+  h4.info,
+  h4.neutral {
+    margin: 0 -1.2rem;
+    padding: 0.8rem 1.2rem 0.8rem 4rem;
+    font-weight: 700;
+    font-style: normal;
+    color: #fff;
+
+    &::before {
+      position: absolute;
+      left: 1.2rem;
+      font-size: 2rem;
+
+      .remix-icon
+    }
+
+    &.warning {
+      background-color: #f3ac55;
+      color: #000;
+
+      &::before {
+        color: #000;
+        content: @icon-error-warning;
+      }
+    }
+
+    &.error {
+      background-color: #eb5949;
+
+      &::before {
+        content: @icon-error-warning;
+      }
+    }
+
+    &.info {
+      background-color: #4496f7;
+
+      &::before {
+        content: @icon-information;
+      }
+    }
+
+    &.neutral {
+      background-color: #101828;
+
+      &::before {
+        content: @icon-double-quotes-l;
+      }
+    }
   }
 
   *:last-child {
```

Comment:
```suggestion
  p:last-child {
```

This fixes this bug on Firefox:

<img width="718" alt="Screenshot 2022-02-20 at 17 35 37" src="https://user-images.githubusercontent.com/9582/154853421-348c14cb-5370-42fd-b2bd-f7f19107157b.png">

Reply:
@josevalim oh cool, also works for safari TIL